jws/jws.go

```diff
@@ -36,8 +36,7 @@ type JWSPayload any
 
 // Base64UrlEncode returns the base64url encoded header.
 func (j Header) Base64UrlEncode() string {
-	jsonHeader := map[string]string{"alg": j.ALG, "kid": j.KID}
```

Comment:
@mistermoe why was this here originally? I ripped it out

Reply:
no idea man 🙈 . i think it may have been from when i originally wrote this as a function instead of a receiver method